sklearn/isotonic.py

```diff
@@ -189,6 +189,10 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     increasing_ : bool
         Inferred value for ``increasing``.
 
+    See Also
+    --------
+    LinearRegression : Ordinary least squares Linear Regression.
+
```

Comment:
Maybe we could also mention `HistGradientBoostingRegressor` because it is non-parametric (like `IsotonicRegression` and contrary to `LinearRegression`) and accepts monotonicity constraints and could therefore be considered a multivariate alternative to `IsotonicRegression`.

Reply:
It would also be worth mentioning the `isotonic_regression` function here.